<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBenchmark</class>
 <widget class="QDialog" name="CBenchmark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Benchmark</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_source">
     <property name="title">
      <string>Source</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_source">
        <item>
         <widget class="QLabel" name="label_file">
          <property name="text">
           <string>File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_openfile">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_source_2">
          <property name="title">
           <string>Dummy generator</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QGridLayout" name="gridLayout_source">
             <item row="3" column="1">
              <widget class="QPushButton" name="pushButton_generatedummy">
               <property name="text">
                <string>Generate</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_sequencelength">
               <property name="text">
                <string>Sequence length</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_chanceone">
               <property name="text">
                <string>Chance of 1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_dummylength">
               <property name="currentIndex">
                <number>5</number>
               </property>
               <item>
                <property name="text">
                 <string>512K</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1M</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2M</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4M</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8M</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>16M</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>32M</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>64M</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>128M</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBox_dummyonechance">
               <property name="suffix">
                <string>%</string>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_filesize">
          <property name="text">
           <string>0KB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_settings">
     <item>
      <widget class="QGroupBox" name="groupBox_settings">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_settings">
          <item row="1" column="0">
           <widget class="QLabel" name="label_passescount">
            <property name="text">
             <string>Passes count</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_lybraryaddresslength">
            <property name="text">
             <string>Library address length</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_benchmarktype">
            <property name="text">
             <string>Benchmark type</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_benchmarktype">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>compress and decompress</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>compress</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>decompress</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_passescount">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBox_libraryaddresslength">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_results">
       <property name="title">
        <string>Results</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_results">
          <item row="0" column="1">
           <widget class="QLabel" name="label_result_compression_r">
            <property name="text">
             <string>0ms</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_result_compression">
            <property name="text">
             <string>Compression</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_result_decompression">
            <property name="text">
             <string>Decompression</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_result_total">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Total</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_result_decompression_r">
            <property name="text">
             <string>0ms</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_result_total_r">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_result_compression_rate">
            <property name="text">
             <string>Compression rate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_result_compression_rate_r">
            <property name="text">
             <string>1.0x</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_footer">
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_logshow">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Lucida Console</family>
        </font>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>logging..</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QProgressBar" name="progressBar_global">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QProgressBar" name="progressBar_local">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_footer">
       <item>
        <spacer name="horizontalSpacer_footer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
